Assets/__Scripts/UI/FirstBootMenu.cs

```diff
@@ -48,38 +55,86 @@ public class FirstBootMenu : MonoBehaviour {
         if (!string.IsNullOrEmpty(posInstallationDirectory))
         {
             directoryField.text = posInstallationDirectory;
+            ValidateQuiet();
         }
 
         directoryCanvas.SetActive(true);
     }
 
-    public void SetDirectoryButtonPressed() {
+    private void SetFromTextbox()
+    {
         string installation = directoryField.text;
-        if (installation == null) {
-            directoryErrorText.text = "Invalid directory!";
+        if (installation == null)
+        {
             return;
         }
         Settings.Instance.BeatSaberInstallation = Settings.ConvertToDirectory(installation);
-        if (Settings.ValidateDirectory(ErrorFeedback))
+    }
+
+    public void SetDirectoryButtonPressed()
+    {
+        SetFromTextbox();
+        if (Settings.ValidateDirectory(ErrorFeedbackWithContinue))
         {
+            SetDefaults();
             FirstBootRequirementsMet();
+            return;
+        }
+        validation.SetValidationState(true, false);
+    }
+
+    public void SetDefaults()
+    {
+        // Auto scale settings for machine
+        if (SystemInfo.processorCount <= 2)
+        {
+            Settings.Instance.Waveform = 0;
+        }
+
+        if (SystemInfo.graphicsMemorySize <= 1024)
```

Comment:
Is it wise to determine performance just by the amount of graphics memory the user has? This could be better off with a simple `Graphics` dropdown with basic options `Performance`, `Balance`, and `Quality`

Reply:
I've still left the gpu memory -> initial selected value, but you can now see it's happening and choose a different value

![image](https://user-images.githubusercontent.com/534069/101216951-567dea00-3678-11eb-890f-1da895f59df9.png)